src/gui.h

```diff
@@ -295,11 +295,17 @@ class TOGGL_INTERNAL_EXPORT AutotrackerRule {
     AutotrackerRule()
         : ID(0)
     , Terms("")
-    , ProjectName("") {}
```

Comment:
While this is not a big deal, for a small improvement in readability and reduction in number of lines of code I suggest using inline member initializers instead of initializing in the constructor.
That means instead of listing members in the constructor like this:
`    , ProjectName("")`
You would set the initial value right in the member declaration (below) like this:
`    std::string ProjectName { "" };`

Reply:
The constructor then doesn't even have to be declared at all but for clarity sake it's probably better to have it.